tests/test_coverage.py

```diff
@@ -142,7 +145,14 @@ def test_gather_providers_content_cafe(self):
         )
 
         # The OCLC provider was already configured; now there is also a ContentCafeCoverageProvider.
-        provider = IdentifierResolutionCoverageProvider(self._default_collection)
+
+        # Instantiating a real ContentCafeAPI will make an HTTP
+        # requests to get a WSDL file, and we don't want that in a
+        # unit test.
+        provider_kwargs = {
+            ContentCafeCoverageProvider : dict(api=MockContentCafeAPI())
+        }
+        provider = IdentifierResolutionCoverageProvider(self._default_collection, provider_kwargs=provider_kwargs)
```

Comment:
This is unrelated to everything else in this branch. I ran the test suite while eating lunch (no wifi) and discovered that this test was making a real HTTP request to get the WSDL file from Content Cafe.